polkadot/node/subsystem-bench/src/core/configuration.rs

```diff
@@ -138,6 +143,11 @@ pub struct TestAuthorities {
 	pub keyring: Keyring,
 	pub validator_public: Vec<ValidatorId>,
 	pub validator_authority_id: Vec<AuthorityDiscoveryId>,
+	pub validator_babe_id: Vec<AuthorityId>,
+	pub validator_assignment_id: Vec<AssignmentId>,
```

Comment:
It fails to compile because they are different types and it doesn't have `into` or `try_into` between them, the content is actually the same but I didn't find an easy way to convert between them.